content/projects/troglio.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Troglio
+repo: Troglio/troglio
+homepage: https://troglio.com/
+twitter: try_troglio
+opensource: "No"
+typeofcms: "Git-based"
+supportedgenerators:
+  - All
+description: Turn Trello into a CMS
+images:
+  - path: /images/troglio-1.png
+  - path: /images/troglio-2.gif
+  - path: /images/troglio-3.gif
+---
+
+# Power your apps and websites from Trello
+
+**Troglio** turns any Trello board into a headless **CMS** : connect any site or app to manage its content from Trello cards.
+
+Writers can write using **markdown**, **TOML** or the Custom Fields Power-up to author their content from anywhere, **even offline !**
+
+Developers can use data as they please without being tied to a provider or a database.
+
+This results in amazing collaborations between authors and developers offering literally no limits to what can be built.
+
+Read our docs at [https://troglio.com/guide](https://troglio.com/guide).
+
+# See  Troglio in action
```

Comment:
Since your uploaded images are displayed at the top, the actual content gets lost because of the additional images underneath. I'd say remove the "troglio in action" section and link to a similar guide instead.